integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/examples/AllExamples.kt

```diff
@@ -99,6 +101,11 @@ val EXAMPLES: ArrayList<Triple<String, @Composable () -> Unit, String?>> =
         Triple("Text Inval", { TextResizingTest() }, TextResizingTestDoc.javaClass.name),
         Triple("Shared Customization", { ModuleExample() }, ModuleExampleDoc.javaClass.name),
         Triple("Variable Modes", { VariableModesTest() }, VariablesTestDoc.javaClass.name),
+        Triple(
+            "State Customizations",
+            @TargetApi(28) { StateCustomizationsTest(Clock.systemDefaultZone()) },
```

Comment:
Does this mean that the test will behave differently depending on your time zone?

Reply:
Yes. Cause using a different time zone rather than the system default, it will show a different time from the device time.